.drone.yml

```diff
@@ -20,7 +20,7 @@ pipeline:
       event: [push]
     image: registry.usw.co/cloud/node-12-puppeteer:03228dd2d438979e8973ec16351c4d7bc88f552e
     commands:
-      - npx lerna run build
+      - npx lerna run build $(for i in $(npx lerna changed); do echo "--scope $i"; done | tr '\n' ' ') --include-filtered-dependents --include-filtered-dependencies
```

Comment:
Depending on how much you care about stuff like this, you can replace the for loop with `sed`:

```
npx lerna changed | sed 's/^/--scope /' | tr '\n' ' '
```